i18n(ko-KR): add `caisy.mdx` & `sanity.mdx` & `sitecore.mdx`. (#6427)
withastro/docs: src/content/docs/ko/guides/cms/sitecore.mdx

---
title: Sitecore Experience Manager & Astro
description: 'Sitecore를 CMS로 사용하여 프로젝트에 콘텐츠를 추가하세요.'
type: cms
stub: true
service: Sitecore XM
i18nReady: true
---

> [Sitecore 경험 관리자 (XM)](https://www.sitecore.com/products/experience-manager)는 ASP.NET을 기반으로 구축된 엔터프라이즈급 콘텐츠 관리 시스템입니다.

Reviewer: ```suggestion
[Sitecore Experience Manager (XM)](https://www.sitecore.com/products/experience-manager)는 ASP.NET을 기반으로 구축된 엔터프라이즈급 콘텐츠 관리 시스템입니다.
```

서비스 명칭은 번역할 필요가 없을 것 같습니다 :)

## 시작하기

> 1. [Sitecore 헤드리스 웹사이트 만들기](https://doc.sitecore.com/xp/en/developers/sxa/103/sitecore-experience-accelerator/create-a-headless-tenant-and-site.html)는 Sitecore의 공식문서를 따르세요.

Reviewer: ```suggestion
1. Sitcore의 공식 문서에 따라 [Sitecore Headless 웹사이트를 만드세요](https://doc.sitecore.com/xp/en/developers/sxa/103/sitecore-experience-accelerator/create-a-headless-tenant-and-site.html).
```

> 2. 터미널에서 다음의 프로젝트 초기 명령어를 실행하세요. 

Reviewer: ```suggestion
2. 터미널에서 다음 프로젝트 초기 명령어를 실행하세요. 
```

  ```shell
  npx @astro-sitecore-jss/create-astro-sitecore-jss@latest
  ```
3. 터미널의 지침에 따라 프로젝트를 생성하세요.

## 커뮤니티 자료

> - [Astro 용 Sitecore Javascript 소프트웨어 개발 키트](https://github.com/exdst/jss-astro-public)

Reviewer: ```suggestion
- Github의 [Astro용 Sitecore JavaScript 소프트웨어 개발 키트](https://github.com/exdst/jss-astro-public)
```

- [Astro를 사용한 Sitecore 소개](https://exdst.com/posts/20231002-sitecore-astro)

> - [첫 번째 Sitecore Astro 프로젝트 시작하기](https://exdst.com/posts/20240103-first-sitecore-astro-project)

Reviewer 2: ```suggestion
- [첫 번째 Sitecore Astro 프로젝트 시작하기](https://exdst.com/posts/20240103-first-sitecore-astro-project)

```
